templates/commands/render/action.go

```diff
@@ -47,29 +47,27 @@ type walkAndModifyVisitor func([]byte) ([]byte, error)
 //
 // If the visitor returns modified file contents for a given file, that file
 // will be overwritten with the new contents.
-func walkAndModify(ctx context.Context, rfs renderFS, scratchDir string, relPaths []model.String, v walkAndModifyVisitor) error {
+//
+// paths are expected to be cleaned already from processPaths before being
+// passed into processGlobs.
```

Comment:
I was thinking of a comment like "rawPaths is a list of globs"

Reply:
Oh, I see. It could also be go template strings or non-glob strings. What about:
"rawPaths is a list of path strings that need to be processed (processPaths, processGlobs) before walking through."

Reply:
SGTM. Maybe "will be" rather than "need to be".

Reply:
Updated comment, rewrapped.